app/views/hub/organizations/index.html.erb

```diff
@@ -11,59 +11,25 @@
 
     <div class="grid">
       <div class="grid__item width-one-half">
-        <% if @coalitions.count > 0 %>
-          <p><%= t(".organizations_by_coalition") %></p>
-
-          <% @coalitions.each do |coalition| %>
-            <div class="grid-flex space-between center-aligned spacing-above-10">
-              <h2 class="spacing-below-0"><%= coalition.name %></h2>
-              <% unless coalition.organizations.count.zero? %>
-                <div class="help-text">Capacity</div>
-              <% end %>
-            </div>
+        <% @organizations_by_coalition.each do |coalition, organizations| %>
```

Comment:
I think I added a bit of view complexity in terms of determining whether to display "Independent Organizations" or nothing for the title, but gained some data simplicity in that we aren't dealing with data from multiple queries with different eager loaded tables that act differently.

Reply:
Makes sense!